### ISSUE TITLE
[Feature] thumbnails in fractions of a second

### ISSUE THREAD

**Issue by lukepighetti** [rurico/flutter_video_compress#62]
Hi there, 

Please allow us to make thumbnails in fractions of a second. If I have a 7 second video and I need 15 thumbnails this is currently not possible. The case in question is creating a filmstrip for a video-trimming workflow. Please support milliseconds or better yet, `Duration`

**Comment by festapp86** [comment]
Yes, that is important for me, too! Is this a limitation of the flutter plugin or something on the OS side? 

At the moment I receive 3 identical thumbnails, if I am calling the method VideoThumbnail.thumbnailFile(...) within the same second.

`    final path2humbnailInt8List = await VideoThumbnail.thumbnailFile(
      video: widget.videoPath,
      thumbnailPath: (await getTemporaryDirectory()).path,
      imageFormat: ImageFormat.JPEG,
      timeMs: i * timeMsWhenThumbnailsAreTaken,
      maxHeightOrWidth: 0,
      // the original resolution of the video
      quality: 100,
    );`
